src/pages/news/index.en.js

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+import { graphql } from "gatsby"
+
+
+import SEO from '../../components/SEO'
+import Navigation from '../../components/Navigation'
+import Layout from '../../components/Layout'
+
+
+import Container from '../../components/Container'
+import Footer from '../../components/Footer'
+
+import PostLink from "../../components/PostLink"
+
+const IndexPage = ({
+  location,
+
+  data: {
+    allMarkdownRemark: { edges }
+  }
+
+}) => (
+  <Layout>
+    <SEO title="Signalen - Latest news" />
+          <Container>
+            <Navigation as="nav" location={location} />
+          </Container>
+
+        <Container>
+              <h2>Latest news</h2>
+              <p>{
+                edges.filter(edge => !!edge.node.frontmatter.date) // You can filter your posts based on some criteria
+                  .map(edge => <PostLink key={edge.node.id} post={edge.node} />)
+              }</p>
+        </Container>
+          <Footer />
+      </Layout>
+)
+
+export default IndexPage
+
+export const pageQuery = graphql`
+  query {
+    allMarkdownRemark(sort: { order: DESC, fields: [frontmatter___date] } filter: {
+      frontmatter: { lang: { eq: "en" } }
+    }) {
+      edges {
+        node {
+          id
+          excerpt(pruneLength: 250)
+          frontmatter {
+            date(formatString: "MMMM DD, YYYY", locale: "en")
```

Comment:
Just checking, this date string is different from the one in index.js. Are they meant to be the same?

Reply:
Yes, it is a formating string that outputs the name of the month, the nr of the day of the month followed by the year in full. The locale is needed to put the month name in the correct language

Reply:
Yeah, my comment wasn't really about the locale, but that here it is `MMMM DD, YYYY` and in the other `DD MMMM YYYY`  so not only is the order different, one also has a comma and the other not.